htdocs/comm/propal/card.php

```diff
@@ -1889,7 +1889,7 @@
 
 		$thirdparty = $soc;
 		$discount_type = 0;
-		$backtopage = urlencode($_SERVER["PHP_SELF"].'?socid='.$thirdparty->id.'&action='.$action.'&origin='.GETPOST('origin').'&originid='.GETPOST('originid'));
+		$backtopage = $_SERVER["PHP_SELF"].'?socid='.$thirdparty->id.'&action='.$action.'&origin='.GETPOST('origin').'&originid='.GETPOST('originid');
```

Comment:
urlencode must be added at least around the param origin and originid because this 2 params are coming from user input.

Reply:
@eldy it's ok, thanks